commands/meme.js

```diff
@@ -0,0 +1,43 @@
+const Augur = require("augurbot"),
+    u = require("../utils/utils");
+
+const Module = new Augur.Module()
+    .addCommand({
+        name: "meme",
+        description: "Creates a meme, put an image URL for you background and then put the text you want along the bottom. Or put the image source afterwards. Who am I to judge?",
+        process: (msg) => {
```

Comment:
Adding in Augur's `suffix` parameters, for line 11.
```suggestion
        process: (msg, suffix) => {
```